app/src/androidTest/java/ch/epfl/culturequest/backend/artprocessingtest/GeneralDescriptionApiTest.java

```diff
@@ -69,7 +76,10 @@ public void getArtDescriptionReturnsCorrectDataWhenPainting(){
         assertThat(artDescription.getArtist(), is("Leonardo da Vinci"));
         assertThat(artDescription.getYear(), is("1517"));
         assertThat(artDescription.getCity(), is("Paris"));
-        assertThat(artDescription.getCountry(), is(nullValue()));
+        assertThat(artDescription.getCountry(), is("France"));
+
+        //todo: investigate Wikipedia doesn't correctly parse the country for Mona Lisa.
```

Comment:
Has this TODO been investigated?

Reply:
Not yet, but it was more out of curiosity than a real need for the app. Anyway, open ai should recover the missing data.